clean_files.txt

```diff
@@ -53,6 +53,7 @@ completion/available/pipx.completion.bash
 completion/available/rustup.completion.bash
 completion/available/vault.completion.bash
 completion/available/sdkman.completion.bash
+completion/available/notify-send.completion.bash
```

Comment:
Unrelated to this PR, but shouldn't we list files in alphabetical order here? Otherwise it's going to get hard to maintain that list as we keep adding more files...

Reply:
I agree

Reply:
@nwinkler Done